e2e/helpers/pages/admin/tags/TagsPage.ts

```diff
@@ -0,0 +1,44 @@
+import {Locator, Page} from '@playwright/test';
+import {AdminPage} from '../AdminPage';
+
+export class TagsPage extends AdminPage {
+    readonly pageContent: Locator;
+    readonly tagList: Locator;
+    readonly tagListRow: Locator;
+
+    readonly tabs: Locator;
+    readonly activeTab: Locator;
+    readonly newTagButton: Locator;
+
+    readonly emptyStateTitle: Locator;
+    readonly emptyStateAction: Locator;
+
+    readonly loadingPlaceholder: Locator;
+
+    constructor(page: Page) {
+        super(page);
+
+        this.pageUrl = '/ghost/#/tags';
+        this.pageContent = page.getByTestId('tags-page');
+        this.tagList = page.getByTestId('tags-list');
+        this.tagListRow = this.tagList.getByTestId('tag-list-row');
+
+        this.tabs = page.getByTestId('tags-header-tabs');
+        this.activeTab = this.tabs.locator('[data-state="active"]');
+        this.newTagButton = page.getByRole('link', {name: 'New tag'});
+
+        this.emptyStateTitle = this.pageContent.getByRole('heading', {name: 'Start organizing your content'});
+        this.emptyStateAction = this.pageContent.getByRole('link', {name: 'Create a new tag'});
```

Comment:
since we use button in other places we should here too.

```suggestion
this.createNewTagButton = this.pageContent.getByRole('link', {name: 'Create a new tag'});
```

Reply:
We're already using the `newTagButton` in the test that I think you are referring to? This is a different button that only shows in the empty state. We could remove it, though I don't think it adds much value.

Reply:
Ah, okay, I see. 

Even though perhaps it's more consistent I also find it a bit confusing. I wouldn't know how to tell them apart without looking at both the locator definition and the UI. The `emptyStateAction` name more clearly conveys the semantic meaning of that button to me and it also avoids that conflict. 

I guess you could call it the `emptyStateActionButton`, but that just gets wordy 😅